meinberlin/apps/newsletters/models.py

```diff
@@ -35,6 +36,7 @@ class Newsletter(UserGeneratedContentModel):
             "When adding images, please ensure to "
             "set the width no larger than 650px."
         ),
```

Comment:
since new migrations get generated for the validators, can you move the help_text to NewsletterForm? Then we won't need to generate migrations again every time the help text changes.

Reply:
is done in #5412 already